src/modules/uORB/uORBDeviceMaster.cpp

```diff
@@ -156,10 +150,6 @@ int uORB::DeviceMaster::advertise(const struct orb_metadata *meta, bool is_adver
 			}
 
 		} else {
-			if (is_advertiser) {
-				node->mark_as_advertised();
```

Comment:
px4_open() -> DeviceNode::open() This has already been done and there is no need to deal with it again.